test/helpers/cilium.go

```diff
@@ -201,6 +204,19 @@ func (c *Cilium) GetEndpointsIds() (map[string]string, error) {
 	return endpoints.KVOutput(), nil
 }
 
+//GetEndpointsIdentityIds returns a mapping of a Docker container name to to
+//its corresponding endpoint identity ID, and an error if the list of endpoints
+//cannot be retrieved via the Cilium CLI.
+func (c *Cilium) GetEndpointsIdentityIds() (map[string]string, error) {
+	// cilium endpoint list -o jsonpath='{range [*]}{@.container-name}{"="}{@.id}{"\n"}{end}'
+	filter := `{range [*]}{@.container-name}{"="}{@.identity.id}{"\n"}{end}`
+	endpoints := c.Exec(fmt.Sprintf("endpoint list -o jsonpath='%s'", filter))
+	if !endpoints.WasSuccessful() {
+		return nil, fmt.Errorf("Can't get endpoint list: %s", endpoints.CombineOutput())
```

Comment:
Do not use upper-case within error messages. C.f. https://github.com/golang/go/wiki/CodeReviewComments#error-strings

For consistency, please use "cannot" instead of "can't". This will make searching logs easier.